### ISSUE TITLE
Extend "Hosting your own index" guide to provide an example for `pypiserver` use

### ISSUE THREAD

**Issue by chrysle** [pypa/packaging.python.org#1467]
I believe `pypiserver` is a simple and efficient way to get an own index server running, and should deserve a recommendation and usage example. The guide is rather thin, in any case.

**Comment by webknjaz** [comment]
I don't think that one is popular. I'm mostly hearing of `devpi` and I also used `dumb-pypi` for static sites in the past. Also, PyPUG is probably not the right place for putting documentation of third party projects. We can link a few possibilities, but that's it.

**Comment by chrysle** [comment]
> I don't think that one is popular.

Don't know how to assess that properly, but it's easily configurable and I thought it might be worth a paragraph.

For example, Pipenv uses it for its test suite and the [Hitchhiker's Guide](https://docs.python-guide.org/shipping/packaging/) introduces it.

**Comment by webknjaz** [comment]
Okay.. It might be somewhat popular, it seems:
- https://pypistats.org/packages/pypiserver
- https://pypistats.org/packages/devpi
- https://pypistats.org/packages/dumb-pypi

We use `devpi` in the integration tests for https://github.com/pypa/gh-action-pypi-publish, by the way.

But I still don't think that we should hosts project-specific docs within PyPUG.

**Comment by chrysle** [comment]
> But I still don't think that we should hosts project-specific docs within PyPUG.

Then I see only the solution that never fails: We must integrate `pypiserver` into PyPA!

No, seriously, I wasn't aware of that guideline. We could add a paragraph comparing some of the most popular solutions, then... ?

**Comment by webknjaz** [comment]
I think it's common sense — why would we want to maintain (a potentially outdated/incorrect) copy somebody else's docs? It's a duplication of effort.

**Comment by chrysle** [comment]
I just revisited #1366 and #1367. What do you think about those?
